generateconfig.py

```diff
@@ -138,42 +138,42 @@ def generate_config():
             if password == password2:
                 break
             else:
-                print "Passwords did not match. Try again"
+                print("Passwords did not match. Try again")
 
-        secret_key = raw_input('CHN Server secret key [""]: ')
-        deploy_key = raw_input('CHN Server honeypot deployment key [""]: ')
+        secret_key = input('CHN Server secret key [""]: ')
+        deploy_key = input('CHN Server honeypot deployment key [""]: ')
 
-        server_base_url = raw_input('Server base url ["{}"]: '.format(default_base_url))
+        server_base_url = input('Server base url ["{}"]: '.format(default_base_url))
         if server_base_url.endswith('/'):
             server_base_url = server_base_url[:-1]
 
         default_honeymap_url = '{}:3000'.format(server_base_url)
-        honeymap_url = raw_input('Honeymap url ["{}"]: '.format(default_honeymap_url))
+        honeymap_url = input('Honeymap url ["{}"]: '.format(default_honeymap_url))
         if honeymap_url.endswith('/'):
             honeymap_url = honeymap_url[:-1]
 
-        mail_server = raw_input('Mail server address ["localhost"]: ')
-        mail_port = raw_input('Mail server port [25]: ')
+        mail_server = input('Mail server address ["localhost"]: ')
+        mail_port = input('Mail server port [25]: ')
 
-        mail_tls = raw_input('Use TLS for email?: y/n ')
+        mail_tls = input('Use TLS for email?: y/n ')
         while mail_tls not in ['y', 'n']:
-            mail_tls = raw_input('Please y or n ')
+            mail_tls = input('Please y or n ')
 
-        mail_ssl = raw_input('Use SSL for email?: y/n ')
+        mail_ssl = input('Use SSL for email?: y/n ')
         while mail_ssl not in ['y', 'n']:
-            mail_ssl = raw_input('Please y or n ')
+            mail_ssl = input('Please y or n ')
 
-        mail_username = raw_input('Mail server username [""]: ')
+        mail_username = input('Mail server username [""]: ')
         mail_password = getpass('Mail server password [""]: ')
 
-        default_mail_sender = raw_input('Mail default sender [""]: ')
+        default_mail_sender = input('Mail default sender [""]: ')
 
-        log_file_path = raw_input('Path for log file ["{}"]: '.format(default_log_path))
+        log_file_path = input('Path for log file ["{}"]: '.format(default_log_path))
 
-        mongo_host = raw_input('MongoDB hostname ["localhost"]: ')
-        mongo_port = raw_input('MongoDB port[27017]: ')
-        hpfeeds_host = raw_input('HPFeeds hostname ["localhost"]: ')
-        hpfeeds_port = raw_input('HPFeeds port[10000]: ')
+        mongo_host = input('MongoDB hostname ["localhost"]: ')
+        mongo_port = input('MongoDB port[27017]: ')
+        hpfeeds_host = input('HPFeeds hostname ["localhost"]: ')
```

Comment:
Since we reference the 'owner' field in https://github.com/CommunityHoneyNetwork/CHN-Server/pull/150/files#diff-52eb648c916ba4bf13cbee2affce1910R65 we should include it as a config item. 

I'm also starting to question why the server needs hpfeeds at all if it's only reading from the database, the database is being fed by mnemosyne,  and there's a separate process for hpfeeds3.